mne/beamformer/tests/test_rap_music.py

```diff
@@ -212,3 +212,15 @@ def test_rap_music_picks():
     noise_cov = mne.read_cov(fname_cov)
     dipoles = rap_music(evoked, forward, noise_cov, n_dipoles=2)
     assert len(dipoles) == 2
+
+
+@testing.requires_testing_data
+def test_rap_music_trap():
```

Comment:
How long does this take to run? Ideally tests will take < 1 sec

Reply:
It should not be longer than the previous tests. Not sure whether they are under the second.  The difference between RAP and TRAP is just one array manipulation where we remove one column in an array at each iteration.